imageregistry/v1/types.go

```diff
@@ -171,6 +171,11 @@ type ImageRegistryConfigStorageS3 struct {
 	// registry.
 	// +optional
 	CloudFront *ImageRegistryConfigStorageS3CloudFront `json:"cloudFront,omitempty" protobuf:"bytes,6,opt,name=cloudFront"`
+	// virtualHostedStyle enables using S3 virtual hosted style bucket paths with
+	// a custom RegionEndpoint
+	// Optional, defaults to false.
+	// +optional
+	VirtualHostedStyle bool `json:"virtualHostedStyle" protobuf:"varint,4,opt,name=virtualHostedStyle"`
```

Comment:
unless this is explicitly what s3 calls it, i'd vote for `VirtualHostedPaths` or "VirtualHostedFormat` over `Style`

Reply:
https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html#virtual-hosted-style-access